lexibank_lieberherrkhobwa.py

```diff
@@ -1,63 +1,88 @@
-from collections import OrderedDict
+# encoding:utf-8
+
+"""
+Module for installing the Lieberherr and Bodt (2017) dataset.
+"""
 
 from clldutils.misc import slug
 from clldutils.path import Path
 from clldutils.text import split_text, strip_brackets
-from pylexibank.dataset import NonSplittingDataset as BaseDataset
+from pylexibank.dataset import NonSplittingDataset
```

Comment:
Okay, now you go for a NonSplittingDataset, but why? This means you will ignore the form after you split them. Is this what you intend? If you look up, a NOnSplittingDataste is defined simply by overriding one function: the split_forms, if I remember correctly.

Reply:
It was a leftover from your code, indeed it makes no sense to load the non-splitting one and override `split_forms` with a splitting function.

Reply:
Ok, that shows how dangerous it is: if you split the value manually, add the form manually, the value will be split again, and then, since segments are taken by what you pass, they will be added twice, this leads to duplicates in the output. This also shows why it is so important that every lexibanker has a much closer look at lexibank code and also looks in fact at the results (!), as they would easily show these problems. E.g., check for the way in which values are split in fact.

Reply:
I see, I will adapt accordingly.

I would however vote for changing this behavior. My preference would be for the `.add_*` methods to write to disk *exactly* the value they receive, without holding the hand of the lexibank developer. Checks for validity and consistency are necessary, of course, but I'd prefer them to take place in an explicit call and after the database creation/installation, in an independent step of the workflow.

Reply:
That's why I initiated the discussion last week, it is getting dangerous, that's why I was re-working on all datasets I am directly involved for the last days.